vaadin-testbench-unit/src/main/java/com/vaadin/testbench/unit/ComponentWrapPackages.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.testbench.unit;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to use to scan given packages for component wrappers outside the
+ * default "com.vaadin.flow.component`.
```

Comment:
minor: package has double quote as prefix and backtick as suffix. Maybe we can use `@code` or `@literal` to wrap the package name.
```suggestion
 * default {@code com.vaadin.flow.component}.
```